src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -398,7 +410,8 @@ export const getL1quote = async (
             slippageBps,
           }): ThorTradeUtxoOrCosmosQuote => {
             const rate = getRouteRate(expectedAmountOutThorBaseUnit)
-            const buyAmountBeforeFeesCryptoBaseUnit = getRouteBuyAmount(quote)
+            const emitAssetAmountCryptoBaseUnit = getEmitAssetAmountCryptoBaseUnit(quote)
```

Comment:
Same thing correct name, which may be indicative of this being wrong, although the memo calc will now be correct, which is the most important thing in this going in asap